packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -1213,24 +1296,31 @@ class Auth {
     /**
      * A wrapper method for commerce-sdk-isomorphic helper: authorizePasswordless.
      */
-    async authorizePasswordless(parameters: AuthorizePasswordlessParams) {
-        const userid = parameters.userid
-        const callbackURI = parameters.callbackURI || this.passwordlessLoginCallbackURI
+    async authorizePasswordless(
+        parameters: AuthorizePasswordlessPublicParams | AuthorizePasswordlessParams
+    ) {
+        // Handle both public and internal calls
+        const isPublicCall = 'userid' in parameters && !('parameters' in parameters)
+        const userid = isPublicCall ? parameters.userid : parameters.parameters?.userid || ''
+        const callbackURI = isPublicCall
+            ? parameters.callbackURI
+            : parameters.parameters?.callbackURI || this.passwordlessLoginCallbackURI
```

Comment:
why would public call and internal call passing params differently? If I were to pass a paramerter, I will have to follow this func signature, not this fuc tryng to adap what is being passed from the outside. 🤔 

Can you give me some examples of this case?

Reply:
Yeah, I'm afraid I don't have the context for public call vs internal call here.

Reply:
The changes were made by Joel so I don't have the context either.
Looking at commit history I see the changes were introduced in this commit:
https://github.com/SalesforceCommerceCloud/pwa-kit/commit/b7b967e11fe40710e4c69319a60a5bd334a0e737

So probably it was a an accidental complexity added to fix build at that moment.
I restored the original API interface

Reply:
That change makes sense I think. 

Before commerce-sdk-isomorphic was pushed, @joeluong-sfcc did make a change to remove the internal APIs so just using the public type sounds right